_data/icons/exorde.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam",

```

Comment:
This is a directory and not a file - so the contentHash in IconDownload is wrong also
```
identify bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam/exorde-logo.png 
bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam/exorde-logo.png PNG 290x401 290x401+0+0 8-bit sRGB 3114B 0.000u 0:00.000

```